core/payment-driver/base/src/driver.rs

```diff
@@ -20,66 +20,88 @@ pub use ya_core_model::payment::local::Network;
 
 #[async_trait(?Send)]
 pub trait PaymentDriver {
+    /// Called by the Identity service to notify the driver that specified
+    /// account is _locked_ / _unlocked_. Identity service holds
+    /// accounts private keys and signs transactions.
     async fn account_event(
         &self,
         _db: DbExecutor,
         _caller: String,
         msg: IdentityEvent,
     ) -> Result<(), IdentityError>;
 
+    /// Gets the balance of the account.
     async fn get_account_balance(
         &self,
         db: DbExecutor,
         caller: String,
         msg: GetAccountBalance,
     ) -> Result<BigDecimal, GenericError>;
 
+    /// Deposits the funds into the driver's supported network. Called by CLI.
     async fn enter(
         &self,
         db: DbExecutor,
         caller: String,
         msg: Enter,
     ) -> Result<String, GenericError>;
 
+    /// Exits the funds outside the driver's supported network (most likely L1).
+    /// Called by CLI.
     async fn exit(&self, db: DbExecutor, caller: String, msg: Exit)
         -> Result<String, GenericError>;
 
     // used by bus to bind service
     fn get_name(&self) -> String;
     fn get_default_network(&self) -> String;
     fn get_networks(&self) -> HashMap<String, Network>;
+
+    /// Tells whether funds are available for spent just after depositing
+    /// on the driver's supported network or the additional unlocking is needed.
```

Comment:
Nope. This flag specifies whether account initialization is needed for **receiving** payments.